src/common/Accessibility.ts

```diff
@@ -363,6 +362,7 @@ class AXNode {
   public isInteresting(insideControl: boolean): boolean {
     const role = this._role;
     if (role === 'Ignored' || this._hidden) return false;
+    if (this._ignored) return false;
```

Comment:
nit: maybe we could also combine this condition check with the check above, since the above check also contains a check for "being ignored or not". WDYT?

Reply:
yeah let's do that :)